autogpt_platform/backend/backend/data/block.py

```diff
@@ -53,6 +53,7 @@ class BlockType(Enum):
     WEBHOOK_MANUAL = "Webhook (manual)"
     AGENT = "Agent"
     AI = "AI"
+    AYRSHARE = "Ayrshare"
```

Comment:
is there a generic term for this or is this a specific need here for brand name? 

i imagine there's other enterprise apis that will make us provision resources on the user's behalf

Reply:
AFAIK it's not about the brand name but about the fact that these blocks are part of a distinct subsystem which allows us to deliver white-label social integrations. I think in the UI it should say "social" but that's a block *category*, not a block type.